src/content/h3/tags/object/projectile/readme.md

```diff
@@ -233,23 +233,21 @@ projection offset (real): Unknown, needs additional research
 
 # Conical Spread
 
-{% figure src="distribution0.jpg" %}
-Pictured: 15 projectiles with a spread angle of 5 degrees and a distribution exponent of 0
-{% /figure %}
+![Perfect distribution](distribution0.jpg "15 projectiles with a spread angle of 5 degrees and a distribution exponent of 0")
 
-{% figure src="distribution1.2.jpg" %}
-Pictured: Now with the distribution exponent set to 1.2 with no other changes
-{% /figure %}
+![Skewed distribution](distribution1.2.jpg "Now with the distribution exponent set to 1.2 with no other changes")
 
-{% alert %}
-Conical spread only works when the "travels instantaneously" flag is enabled and a value other than 0 is set for "maximum range"
-{% /alert %}
+This section can be used for weapons that fire a high volume of projectiles in a single shot, like shotguns: it defines the size and spread of the entire blast and allows more control compared to the error fields in the [weapon](~weapon#error-spread-bloom) tag.
+
+Conical spread requires the "travels instantaneously" flag to be enabled and for the projectile velocity to be of suffiencitly high enough velocity to reach any target (map geo included) in a single tick *within* a desired range, this range is determined by the projectile velocity and still follows the ["maximum range" field](~projectile#detonation).
```

Comment:
Style nit: When referring to specific tag fields by name, we generally write them like `_travels instantaneously_` which italicizes them. This is widely done across the CE pages so I'd like to keep it consistent.